src/components/UI/Modal/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from "react";
```

Comment:
![image](https://user-images.githubusercontent.com/503185/63679843-e18cf500-c824-11e9-802f-75005c87c3b7.png)


It's not pretty at all ^^